cg/cli/reset.py

```diff
@@ -14,25 +19,48 @@ def reset_cmd():
     pass
 
 
-@reset_cmd.command()
-@click.option("-c", "--case_id", help="internal case id, leave empty to process all")
+@reset_cmd.command("observations")
+@ARGUMENT_CASE_ID
 @click.pass_obj
-def observations(context: CGConfig, case_id: Optional[str]):
-    """Reset observation links from an analysis to LoqusDB."""
+def observations(context: CGConfig, case_id: str):
+    """Reset observation links from an analysis to Loqusdb for a specific case."""
+
     status_db: Store = context.status_db
-    observations_uploaded: Iterable[models.Family]
-    if case_id:
-        observations_uploaded = [status_db.family(case_id)]
-    else:
-        observations_uploaded = status_db.observations_uploaded()
+    case: models.Family = context.status_db.family(case_id)
 
-    for case_obj in observations_uploaded:
-        LOG.info("This would reset observation links for: %s", case_obj.internal_id)
+    if not case or case.data_analysis not in LOQUSDB_SUPPORTED_PIPELINES:
+        LOG.error("Invalid case ID. Retrieving available cases for Loqusdb reset.")
+        uploaded_observations: Query = status_db.observations_uploaded()
+        if not uploaded_observations:
+            LOG.info("There are no available case uploaded observations")
+        else:
```

Comment:
```suggestion
            LOG.info("There are no available case uploaded observations")
            raise click.Abort()
```
It we raise here we can remove the else and make the next block more shallow